client/src/components/jobPosting/TimespanPicker.js

```diff
@@ -45,21 +53,21 @@ export class TimespanPicker extends Component {
   }
 
   async handleShowFromChange(date) {
-    if (date !== undefined && (date === null || !this.state.showTo || date.isBefore(this.state.showTo))) {
-      if (date === null) {
-        await this.setState({
-          showFrom: null,
-          showTo: null
-        })
-        this.props.addShowFrom(null)
-      } else {
-        await this.setState({
-          showFrom: date,
-          error: ''
-        })
-        const formattedDate = date.toLocaleString()
-        this.props.addShowFrom(formattedDate)
-      }
+    if (!date) {
+      await this.setState({
+        showFrom: null,
+        showTo: null
+      })
+      this.props.addShowFrom(null)
+    }
+
+    if (!this.state.showTo || date.isSameOrBefore(this.state.showTo)) {
+      await this.setState({
+        showFrom: date,
+        error: ''
+      })
+      const formattedDate = date.toLocaleString()
```

Comment:
When user is creating a new posting and clicks to empty the "from" field then app crashes and this line throws error "Cannot read property 'toLocaleString' of null".